src/sentry/integrations/slack/unfurl/discover.py

```diff
@@ -43,6 +44,13 @@ def unfurl_discover(data, integration, links: List[UnfurlableUrl]) -> UnfurledUr
 
         params = link.args["query"]
         query_id = params.get("id", None)
+        user_id = params.get("user", None)
```

Comment:
You may be able to avoid doing this passing the user to the internal API request instead, and all this logic will be handled for you.